docs/Part02ConcurrentSMTesting.md

```diff
@@ -50,10 +50,14 @@ Let’s say that the `get`s returned `1` and `3` respectively. Is this a correct
 
 What if the `get`s both returned `3`? That’s also correct and witnessed by the second interleaving `< incr 1, incr 2, get, get >`. When we can find a sequential interleaving that supports the outcome of a concurrent execution we say that the concurrent history linearises.
 
-If the second `get` returned `1` or `2` however it would be a non-linearising outcome. We can see visually that the second `get` happens after both `incr`. When can this occur? Imagine if `incr` is implemented by first reading the current value then storing the incremented value, in that case there can be a race where the `incr`s overwrite each other.
+If the `get` on the third thread returned `1` or `2` however, then it would be a non-linearisable outcome. We can see visually that that `get` happens after both `incr`, so no matter where we choose to place the red crosses on the `incr`s the effects will happen before that `get` so it must return `3`. Is it even possilbe that `1` or `2` are returned? It’s, imagine if `incr` is implemented by first reading the current value then storing the incremented value, in that case there can be a race where the `incr`s overwrite each other.
```

Comment:
```suggestion
If the `get` on the third thread returned `1` or `2` however, then it would be a non-linearisable outcome. We can see visually that that `get` happens after both `incr`, so no matter where we choose to place the red crosses on the `incr`s the effects will happen before that `get` so it must return `3`. Is it even possilbe that `1` or `2` are returned by a real implementation? It’s, imagine if `incr` is implemented by first reading the current value then storing the incremented value, in that case there can be a race where the `incr`s overwrite each other.
```
I think it could be good to stress that it is errors in the real implementation, not due to weird linearisation interleavings.

Reply:
Thanks!